app/helpers/complaints_helper.rb

```diff
@@ -35,4 +35,22 @@ def sort_date(date)
   def sort_status(status_label)
     STATUS_SORT_ORDER[status_label]
   end
+
+  def aria_sort(column)
+    current_sort_direction = current_sort_direction(column)
+    if current_sort_direction.present?
+      "aria-sort=\"#{current_sort_direction}\""
+    end
+  end
+
+  def current_sort_direction(column)
+    if @sort.nil?
```

Comment:
(however, whatever you decide to do, please add a comment that explains what the match is matching on)

Reply:
+1 on adding a comment about the `-columnName`.  We know how that works because we know we're using JSON:API, but this would be really opaque to me otherwise.